config/initializers/clowder_config.rb

```diff
@@ -50,7 +50,7 @@ def self.instance
 
       options["SOURCES_URL"] = ENV["SOURCES_URL"]
       # TODO: update with valid url later
-      options["MQTT_CLIENT_URL"] = ENV["MQTT_CLIENT_URL"] || "mqtt://localhost:1883"
+      options["CLOUD_CONNECTOR_URL"] = ENV["CLOUD_CONNECTOR_URL"] || "mqtt://localhost:1883"
```

Comment:
@hsong-rh The default if missing is mqtt which is not correct for Cloud Controller since they talk over HTTP